README.md

```diff
@@ -20,23 +20,71 @@ cluster on top of bare metal infrastructure using Metal3.
 | v1alpha4      | v1alpha3            | v0.4.X        |
 | v1alpha5      | v1alpha4            | v0.5.X        |
 
-You can find information on how to use this provider with Cluster API and
-clusterctl in the [getting-started](docs/getting-started.md).
-
 ## Deploying the metal3 provider
 
 The recommended method is using
 [Clusterctl](https://master.cluster-api.sigs.k8s.io/clusterctl/overview.html).
-Please refer to the [getting-started](docs/getting-started.md) for the
-pre-requisites.
+
+Starting from `v0.5.0` release of Cluster-api-provider-metal3, Baremetal Operator is decoupled
+from Cluster-api-provider-metal3 deployments when deployed via `clusterctl`. For that reason,
+Baremetal Operator will not be installed when initializing the Metal3 provider with clusterctl,
+and its CRDs and controller need to be manually installed. Example flow of installing Metal3
+provider:
+
+1. Install Cluster API core, bootstrap and control-plane providers. This will also install
+  cert-manager if it is not already installed.
+
+    ```shell
+    clusterctl init --core cluster-api:v0.4.4 --bootstrap kubeadm:v0.4.4 \
```

Comment:
since this is going to main branch, should it be `v1.0.0` here and leave it as is in back port of this patch?

Reply:
I misread branches, ^ it is other way around

Reply:
@furkatgofurov7 maybe not now, since we don't have a release with v1beta1 APIs yet

Reply:
We could update this once the release is made, wdyt?

Reply:
This is fine, we have to on the other one (pointing to main branch)